app/domain/authentication/authenticate.rb

```diff
@@ -29,7 +29,6 @@ def call
       audit_success
```

Comment:
what about moving this to the authn controller as well? can we align the logic here?